run_pacta_data_preparation.R

```diff
@@ -314,12 +314,10 @@ invisible(gc())
 
 logger::log_info("Formatting and saving file: \"entity_info.rds\".")
 factset_entity_id__ar_company_id <-
-  readr::read_csv(ar_company_id__factset_entity_id_path, col_types = "c") %>%
-  select(
-    factset_entity_id = "factset_id",
-    ar_company_id = "company_id"
-  ) %>%
-  distinct()
+  pacta.data.preparation::prepare_factset_entity_id__ar_company_id(
+    readr::read_csv(ar_company_id__factset_entity_id_path, col_types = "c")
+  )
```

Comment:
NIT/ style: maybe pipe this for clarity? I always find it a little difficult to read when the first argument of a function is another function with multiple arguments

Reply:
done in all cases